### PR TITLE
support the console repo split

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
i've tested the console actions, working on testing the updated origin actions that dont use asset/console stuff anymore

@danmcp want to go ahead and review the changes?

File: lib/vagrant-openshift/action.rb [hunk 1]
```diff
@@ -174,6 +188,18 @@ def self.run_origin_tests(options)
           b.use TestExitCode
         end
       end
+      
+      def self.run_origin_asset_tests(options)
+        Vagrant::Action::Builder.new.tap do |b|
+          # UI integration tests require the api server to be running
+          b.use RunSystemctl, {:action => "restart", :service => "openshift"}
```
Comment (danmcp) [inline lib/vagrant-openshift/action.rb]: why not just start?

File: lib/vagrant-openshift/plugin.rb [hunk 2]
```diff
@@ -36,6 +36,11 @@ class Plugin < Vagrant.plugin("2")
         require_relative "command/repo_sync_origin"
         Commands::RepoSyncOrigin
       end
+
```
Comment (danmcp) [inline lib/vagrant-openshift/plugin.rb]: Fix whitespace

File: lib/vagrant-openshift/plugin.rb [hunk 3]
```diff
@@ -121,6 +126,11 @@ class Plugin < Vagrant.plugin("2")
         require_relative "command/test_origin"
         Commands::TestOrigin
       end
+
```
Comment (danmcp) [inline lib/vagrant-openshift/plugin.rb]: Fix whitespace

File: lib/vagrant-openshift/plugin.rb [hunk 4]
```diff
@@ -136,6 +146,11 @@ class Plugin < Vagrant.plugin("2")
         require_relative "command/download_artifacts_origin"
         Commands::DownloadArtifactsOrigin
       end
+
```
Comment (danmcp) [inline lib/vagrant-openshift/plugin.rb]: Fix whitespace

File: lib/vagrant-openshift/command/repo_sync_origin_console.rb [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+            o.on("-s", "--source", "Sync the source (not required if using synced folders)") do |f|
+              options[:source] = f
+            end
+
+            o.on("--dont-install", "Don't build and install updated source") do |f|
+              options[:build] = false
+            end
+
+            o.on("--no-images", "Don't build updated component Docker images") do |f|
```
Comment (danmcp) [inline lib/vagrant-openshift/command/repo_sync_origin_console.rb]: Does this option mean anything?

File: lib/vagrant-openshift/command/repo_sync_origin_console.rb [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+            o.on("-c", "--clean", "Delete existing repo before syncing") do |f|
+              options[:clean] = f
+            end
+
+            o.on("-s", "--source", "Sync the source (not required if using synced folders)") do |f|
+              options[:source] = f
+            end
+
+            o.on("--dont-install", "Don't build and install updated source") do |f|
```
Comment (danmcp) [inline lib/vagrant-openshift/command/repo_sync_origin_console.rb]: Does this option mean anything?

File: lib/vagrant-openshift/action.rb (hunk 1 above)
Comment (jwforres) [inline lib/vagrant-openshift/action.rb]: start should be fine, the other thing i debated is whether i should do a stop after the tests have run

File: lib/vagrant-openshift/command/repo_sync_origin_console.rb (hunk 5 above)
Comment (jwforres) [inline lib/vagrant-openshift/command/repo_sync_origin_console.rb]: I need to remove this one

File: lib/vagrant-openshift/command/repo_sync_origin_console.rb (hunk 6 above)
Comment (jwforres) [inline lib/vagrant-openshift/command/repo_sync_origin_console.rb]: so i can have this trigger whether the install asset dependencies step runs

**Pull request merged**